src/test/java/edu/harvard/iq/dataverse/api/IndexIT.java

```diff
@@ -0,0 +1,122 @@
+        uploadMd5File.prettyPrint();
+        assertEquals(CREATED.getStatusCode(), uploadMd5File.getStatusCode());
+   
+        Response response = given()
+                .header(API_TOKEN_HTTP_HEADER, apiToken)
+                .get("/api/admin/index/status");
+        response.prettyPrint();
+        ArrayList emptyList = new ArrayList<>();
+        HashMap data = response.getBody().jsonPath().get("data.contentInDatabaseButStaleInOrMissingFromIndex");
+        System.out.println(data + " " + data.getClass());
+        response.then().assertThat().statusCode(OK.getStatusCode())
+                .body("data.contentInDatabaseButStaleInOrMissingFromIndex.dataverses", CoreMatchers.equalTo(emptyList))
+                .body("data.contentInDatabaseButStaleInOrMissingFromIndex.datasets", CoreMatchers.equalTo(emptyList))
+                .body("data.contentInIndexButNotDatabase.dataverses", CoreMatchers.equalTo(emptyList))
+                .body("data.contentInIndexButNotDatabase.datasets", CoreMatchers.equalTo(emptyList))
+                .body("data.contentInIndexButNotDatabase.files", CoreMatchers.equalTo(emptyList))
+                .body("data.permissionsInDatabaseButStaleInOrMissingFromIndex.dvobjects", CoreMatchers.equalTo(emptyList))
+                .body("data.permissionsInIndexButNotDatabase.permissions", CoreMatchers.equalTo(emptyList));
+        
+        Response getDatasetJsonAfterMd5File = UtilIT.nativeGet(datasetId, apiToken);
+        getDatasetJsonAfterMd5File.prettyPrint();
+        getDatasetJsonAfterMd5File.then().assertThat()
+                .body("data.latestVersion.files[0].dataFile.md5", equalTo("0386269a5acb2c57b4eade587ff4db64"))
+                .body("data.latestVersion.files[0].dataFile.checksum.type", equalTo("MD5"))
+                .body("data.latestVersion.files[0].dataFile.checksum.value", equalTo("0386269a5acb2c57b4eade587ff4db64"));
+
+        int fileId = JsonPath.from(getDatasetJsonAfterMd5File.getBody().asString()).getInt("data.latestVersion.files[0].dataFile.id");
+        Response deleteFile = UtilIT.deleteFile(fileId, apiToken);
+        deleteFile.prettyPrint();
+        deleteFile.then().assertThat()
+                .statusCode(NO_CONTENT.getStatusCode());
+
+        Response deleteDatasetResponse = UtilIT.deleteDatasetViaNativeApi(datasetId, apiToken);
+        deleteDatasetResponse.prettyPrint();
+        deleteDatasetResponse.then().assertThat()
+                .statusCode(OK.getStatusCode());
+
+        Response deleteDataverseResponse = UtilIT.deleteDataverse(dataverseAlias, apiToken);
+        deleteDataverseResponse.prettyPrint();
+        deleteDataverseResponse.then().assertThat()
+                .statusCode(OK.getStatusCode());
+
+        Response deleteUserResponse = UtilIT.deleteUser(username);
+        deleteUserResponse.prettyPrint();
+        assertEquals(200, deleteUserResponse.getStatusCode());
```

Comment:
I go back and forth on cleanup like this. One one hand it's nice not to junk up your dev environment with test data. On the other hand, if the delete fails and the whole test suite fails as a result, it's frustrating. One thing I sometimes do is put the delete calls in but don't assert that they succeed. No need to changes anything here. It's just something that popped in my head.

Reply:
removing the assert tests from the delete calls.